downloader/src/funTest/kotlin/vcs/GitWorkingTreeFunTest.kt

```diff
@@ -72,8 +72,7 @@ class GitWorkingTreeFunTest : StringSpec({
         workingTree.listRemoteBranches().filterNot { it.startsWith("dependabot/") } should containExactlyInAnyOrder(
             "all-repos_autofix_bump",
             "all-repos_autofix_bump-2023-02-05",
-            "main",
-            "pre-commit-ci-update-config"
```

Comment:
FYI, this branch has disappeared and reappeared before, see 8b4e489ddf144c09ee43605b1f7fdf77e2f329b2 and 807c3635b17f5ac40de5413f703edd4eeee5fa68. If this happens again, we should probably consider ignoring it.

Reply:
I've addressed this with: https://github.com/oss-review-toolkit/ort/pull/7157.